root.tf

```diff
@@ -691,6 +717,46 @@ module "s3_vpc_endpoint" {
   )
 }
 
+module "github_oidc_provider" {
+  source      = "./tdr-terraform-modules/identity_provider"
+  audience    = "sts.amazonaws.com"
+  thumbprint  = "6938fd4d98bab03faadb97b34396831e3780aea1"
+  url         = "https://token.actions.githubusercontent.com"
+  common_tags = local.common_tags
+}
+
+module "consignment_api_github_environment" {
```

Comment:
Maybe rename slightly to make clearer what this module is: `github_consignment_api_environment`